enforcer-rules/README.md

```diff
@@ -0,0 +1,89 @@
+# Linkage Checker Enforcer Rule
+
+This Maven enforcer rule verifies that the transitive dependency tree of `pom.xml` does not have
+any [linkage error](../library-best-practices/glossary.md#types-of-conflicts-and-compatibility).
+
+## Class path and dependencySection flag
+
+The value of the dependencySection element determines whether the rule checks the dependencies in
```

Comment:
I think we can now delete "value of the"

Reply:
Deleted.